src/views/components-default/TheDefaultMobileNavigation.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="mew-component--landing-page-menu-mobile">
-    <app-btn-menu height="40" x-large @click.native="isOpen = true" />
+    <app-btn-menu @click.native="openMobileMenu" />
```

Comment:
You also need to change AppBtnMenu implementation (since menu-method prop was removed) in here: https://github.com/MyEtherWallet/MyEtherWallet/blob/v6/update-mobile-menu-ui/src/views/components-wallet/TheWalletSideMenu.vue

Reply:
fixed